ENIGMAsystem/SHELL/Widget_Systems/widgets_mandatory.h

```diff
@@ -23,7 +23,7 @@
 
 #include <string>
 
-#define DEBUG_MESSAGE(msg, severity) ::enigma_user::show_debug_message((std::string) (msg) + " | " __FILE__ ":" + std::to_string(__LINE__), (severity))
+#define DEBUG_MESSAGE(msg, severity) ::enigma_user::show_debug_message((std::string) (msg) + " | " __FILE__ ":" + std::to_string(__LINE__) + "\n", (severity))
```

Comment:
Don't think the \n belongs here as widgets don't need it. It should probably only be in the cli's implementation